docs/installation.md

```diff
@@ -57,11 +57,22 @@ vagrant destroy
 
 ### Quick Setup 2: Docker
 SMACK can also be run in a [Docker][] container. We tested the Dockerfile on
-the following configurations:
+the following configuration:
 
-* Ubuntu 16.04, docker-ce version 18.09.7
-* OS X 10.14.5, Docker Desktop with Docker engine version 18.09.2
-* Windows 10, Docker Desktop with Docker engine version 18.09.2
+* Ubuntu 18.04, Docker version 19.03.6
+* Windows WSL Ubuntu 20.04, Docker Desktop with Docker engine version 20.10.2
+
+
+#### Docker Hub
+
+SMACK's Docker container images can be pulled from Docker Hub directly:
```

Comment:
I think this should say:
The SMACK's Docker container images can be pulled from the Docker Hub directly:

Reply:
"The SMACK's ..." reads strangely.

Reply:
Ok, you are a native speaker :). If what @shaobo-he wrote looks good to you, let's role with that.